src/Homepage/Homepage.scss

```diff
@@ -1,10 +1,66 @@
 //font-family: 'Condiment', cursive;
 //font-family: 'Condiment', cursive;
 //font-family: 'Delius', cursive;
-.main-title{
-  font-size: 4em;
-  letter-spacing: 3px;
-  color: #7ea04d;
-  font-family: 'Condiment', cursive;
-  background-color: #f8efd4;
-}
\ No newline at end of file
+
+  
+  .nav-bar{
+
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+    align-items: center;
+    justify-content: space-between;
+    height: 30vh;
+    background-color: #0a1407;
+
+
+    .main-title{
+      font-size: 4em;
+      letter-spacing: 3px;
+      color: #7ea04d;
+      font-family: 'Condiment', cursive;
+      margin:5%;
+      width: 100%;
+    }
+
+    .search-container{
+      display: inline-flex;
+      width: 25%;
+      
+      .search-bar{
+        border: none;
+        border-bottom: 2px solid #7ea04d;
+        width: 15em;
+        height: 2em;
+      }
+
+      .fa-search{
+        color: #7ea04d;
+        font-size: 1.5em;
+        margin: 2%;
+      }
```

Comment:
Are these nested in navBar? Sweet! I've never used this syntax before. I'll have to look into it a bit more.

Reply:
it is getting sassy!